Sources/ScipioKit/Producer/PIF/FrameworkBundleAssembler.swift

```diff
@@ -0,0 +1,121 @@
+import Foundation
+import TSCBasic
+
+/// A assembler to generate framework bundle
```

Comment:
```suggestion
```

It is already described in name of struct

Reply:
There is an additional doccomment following line. So it's better to remain this line.